wsclient/src/test/java/org/hpccsystems/ws/client/WSPackageProcessTest.java

```diff
@@ -39,12 +55,27 @@ public void ping() throws Exception
         catch (AxisFault e)
         {
             e.printStackTrace();
-            Assert.fail();
+            fail();
         }
         catch (Exception e)
         {
             e.printStackTrace();
-            Assert.fail();
+            fail();
         }
     }
+
+    @Test
+    public void listPackage() throws Exception {
+        List<PackageListMapDataWrapper> packages = client.listPackages("*", targetRoxieName, null);
+        if(packages==null || packages.isEmpty()) {
+            fail("could not retrieve package lists from target cluster : " + targetRoxieName );
```

Comment:
since targetRoxieName could be empty, let's wrap it in single quotes:
fail("could not retrieve package lists from target cluster : '" + targetRoxieName  + "'");